spec/api.test.js

```diff
@@ -0,0 +1,15 @@
+const app = require("../server/app");
+const supertest = require("supertest");
+const request = supertest(app);
+
+it("gets the photos endpoint", async (done) => {
+  const response = await request.get("/photos");
+  expect(response.status).toBe(200);
+  done();
```

Comment:
Did you have trouble with these async/await functions like I did? I had to make some changes to the eslint config before it would accept this syntax.